azurerm/resource_arm_postgresql_server.go

```diff
@@ -372,6 +372,8 @@ func expandAzureRmPostgreSQLServerSku(d *schema.ResourceData) *postgresql.Sku {
 	skus := d.Get("sku").([]interface{})
 	sku := skus[0].(map[string]interface{})
 
+	fmt.Printf("[SKU]: %s", sku)
```

Comment:
_minor_ can we remove this?

Reply:
DOH! Sorry, that was part of my debug code... I forgot to remove it thanks for spotting it! :)